src/Swarrot/Broker/MessageProvider/MessageProviderInterface.php

```diff
@@ -13,6 +13,14 @@
      */
     public function get();
 
+    /**
+     * consume
+     *
+     * @param string $consumerTag Identify the consumer for the rabbitmq-server
```

Comment:
Talking about `rabbitmq-server` in the description of this interface looks wrong to me. This interface is not specific to RabbitMQ.